src/views/Farm/components/Stake/V3Farm.tsx

```diff
@@ -51,9 +51,9 @@ const Stake: React.FC = () => {
 
   const handleOnUnstake = useCallback(
     (nftId: number) => {
-      onWithdraw(nftId, DpiEthRewards)
+      onWithdraw(nftId, props.farm)
```

Comment:
I think since you destructured the param on 19 you can replace instances of `props.farm` with `farm`